src/Microsoft.AspNetCore.Mvc.Core/Internal/ApiBehaviorOptionsSetup.cs

```diff
@@ -36,8 +36,8 @@ IActionResult GetInvalidModelStateResponse(ActionContext context)
                     new BadRequestObjectResult(modelState) :
                     new BadRequestObjectResult(errorDetails);
 
-                result.ContentTypes.Add("application/problem+json");
-                result.ContentTypes.Add("application/problem+json");
+                result.ContentTypes.Add("application/json");
+                result.ContentTypes.Add("application/json");
```

Comment:
This duplicate should be removed, no? I think I raised this with @rynowak earlier, but can't find the comment...